### ISSUE TITLE
Page component title doesn't change with multiple route

### ISSUE THREAD

**Issue by tommypepsi** [Shopify/polaris#309]
# Issue summary
I'm fairly new to react so I might be missing something. I'm using react-router for the routes. Each routes render a page component with a different title but the title stays the same of the initial page.

on the first route I can put a setTimeout and update the title so the title can be changed(at least I know that much), but when I go to the second route with a new page component the title cannot be affected anymore.

## Expected behavior
I expect the page title to be updated when I load a new Page component.

## Actual behavior
The page title is not updated

## Steps to reproduce the problem
With react-router create multiple route with each a Page component with different titles.

## Specifications

- Polaris version: 1.14.2
- React version: 16.3.2
- Browser: chrome
- Device: desktop
- Operating System: windows 10

**Comment by chloerice** [comment]
Hi @tommypepsi, can you share a code snippet of the page that renders the incorrect title, as well as a  snippet of how you've set up your routes? Does the title not update on that particular route regardless of whether it's navigated to from another page or is the first to render?

**Comment by ry5n** [comment]
Without additional information we’re not able to help debug. Happy to re-open if you can provide more information.

**Comment by tommypepsi** [comment]
Hi again. Sorry for the long delay I forgot I posted that here. The problem I had was that I had multiple route and each route had their own Page component with their own title, but when a new route was loaded the title didn't change. I guess the page component parameters are only applied on the first load. I managed to fix this by having a single parent Page Component for all the routes and dynamically changing the title variable with redux when a new route is loaded. I think it would have been more intuitive if I could've had a Page component for each routes, but I found a way so everything's ok in the end.
